# Please don&#39;t start a variable name with &#34;the&#34; (no-the-prefix)

It's just annoying...

## Rule Details

Examples of **incorrect** code for this rule:

```js

const theFoo = "bar";

```

Examples of **correct** code for this rule:

```js

const foo = "bar";

```
